docs/learn/learn-comparisons-avalanche.md

```diff
@@ -0,0 +1,112 @@
+
+## Consensus 
+
+![Avalanche Network](../assets/comparisons/avalanche/avalanche-consensus-protocols.png)
+
+Avalanche consensus uses a family of protocols to achieve security, liveness, and finality. These are known as the Snow* protocols. This group of protocols composed together uses both classical and Nakamoto consensus as well as a Delegated Proof-of-Stake system for its block creators.
+
+The Snow family is a hierarchical collection of systems used to reach finality on Avalanche:
+
+- Slush
+- Snowflake
+- Snowball
+- Avalanche
+- Snowman
+- Slushie
+
+
+Compared to Polkadot, Avalanche uses an asynchronous hybrid system that is based on a classical and Nakomoto approach. Polkadot uses a synchronous hybrid model that combines BABE and GRADNPA, where BABE is the algorithm used to build blocks in a probabilistic way, and GRANDPA is a finality mechanism that uses a deterministic approach to adding blocks to the longest chain. In the end, validators agree to whole chains, rather than single new blocks.
+
+### Snowball 
+
+The snowball protocol is an algorithm that nodes use to come to a consensus. Each node continuously queries x number of validators and takes the majority consensus and adopts it as its own. This method, in normal circumstances, will lead to the network reaching a consensus. The scalability of Snowball is promising, as the number of participants in the network grows, the number of consensus messages being passed around remains the same. Nodes will query no more than 20 nodes at a given time.
+
+### DAG(Directed Acyclic Graph)
+
+[DAGs](https://en.wikipedia.org/wiki/Directed_acyclic_graph) are graphs consisting of vertices and edges. In Avalanche they are used for **partial ordering** of decisions, such as transactions. Vertices point to each other using edges, and when ordered topologically vertices and edges create a sequence. Edges in the case of Avalanche can be conflicting, and nodes will use the snowball algorithm to make decisions about which edges to keep and which to not. 
+
+## Staking Mechanics
+
+Avalanche uses a Delegated Proof-of-Stake mechanism without any slashing. The barrier to entry for staking as a full node validator is 2500 AVAX, and 25 AVAX to become a delegator. With a minimum stake period being two weeks and a maximum period being a year, for both validators and delegators. Validators acquire points for uptime and correctness of their work, and the remuneration of rewards depends on that.
```

Comment:
Might be worth mentioning about Polkadot staking parameters here as well. Also, curious to know what happens after a year? Do the stakers have to re-stake?